frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/projects/ProjectPortlet.java

```diff
@@ -238,6 +241,20 @@ public void serveResource(ResourceRequest request, ResourceResponse response) th
             JSONObject dataForChangeLogs = changeLogsPortletUtilsPortletUtils.serveResourceForChangeLogs(request,
                     response, action);
             writeJSON(request, response, dataForChangeLogs);
+        } else if (PortalConstants.LOAD_OBLIGATIONS_VIEW.equals(action)
```

Comment:
Only `request.setAttribute("inProjectDetailsContext", xxxx);` statement seems to only difference in conditions